armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -11,6 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from _operator import index
+from _ast import Or
```

Comment:
Sorry, I might be blind. But I don't see these imports being used.  

Are these unused imports, or am I missing something obvious?